docs/source/faq.rst

```diff
@@ -494,3 +494,42 @@ There are two kinds of constrained optimizations, one with soft constraints and
 Soft constraints do not have to be satisfied, but an objective function is penalized if they are unsatisfied. On the other hand, hard constraints must be satisfied.
 
 Optuna is adopting the soft one and **DOES NOT** support the hard one. In other words, Optuna **DOES NOT** have built-in samplers for the hard constraints.
+
+How can I parallelize optimization?
+-------------------------------------------------
+
+The variations that can be expected to be parallelized are in the following three cases.
+
+1. Multi-threading parallelization with single node
+2. Multi-processing parallelization with single node
+3. Multi-processing parallelization with multiple nodes
+
+For the case of 1
+================================
+
+Parallelization can be achieved by setting the argument `n_jobs` in the optimize method of the :class:`~optuna.study.Study` class.
+However, since it is multi-threading, the python code will not be faster due to GIL. 
+
+While optimizing, it will be faster in limited situations, such as waiting for other server requests, or waiting for C/C++ processing with numpy, etc., but it is not expected to be faster in other cases.
+
+For more information about 1., see APIReference_.
+
+.. _APIReference: https://optuna.readthedocs.io/en/stable/reference/index.html
+
+For the case of 2
+================================
+
+This can be achieved by using RDB or sqlite.
+However, if you cannot install RDB or sqlite, it is not supported, so if you have a request, please make an Issue.
```

Comment:
I'm afraid that I could not understand this sentence. What do you mean by this sentence?

Reply:
There seemed to be some inconsistencies in the text, which were pointed out by others, and I have responded to correct them.